var/spack/repos/builtin/packages/netgauge/package.py

```diff
@@ -18,3 +18,10 @@ class Netgauge(AutotoolsPackage):
     version('2.4.6', sha256='dc9398e4e042efec70881f2c7074ff18cc5b74bc5ffc4b8a4aaf813b39f83444')
 
     depends_on("mpi")
+
+    def configure_args(self):
+        args = []
+        if self.spec.target.family == 'aarch64':
+            args.append('MPICC=mpicc')
+            args.append('MPICXX=mpicxx')
```

Comment:
I wonder if we shouldn't use `self.spec['mpi'].mpicc` (the absolute path to the mpi wrapper, independent of mpi library) and do this for all platforms...